### PR TITLE
NewCodeEdit

### PR DESCRIPTION
**Pull request by cbezzy**
fixes : multi select, external data search
Add pagination
Move fancy box to assess dir to cater for modernisation project
Remove Standard Price From Code Editor Screen, All Prices In Price
Editor Screen

### CODE REVIEW

File: interface/patient_file/encounter/superbill_custom_full.php [hunk 1]
```diff
@@ -13,662 +13,250 @@
...
-
- <tr<?php if (empty($GLOBALS['ippf_specific'])) echo " style='display:none'"; ?>>
-  <td><?php echo xlt('CYP Factor'); ?>:</td>
-  <td></td>
-  <td>
-   <input type='text' size='10' maxlength='20' name="cyp_factor" value='<?php echo attr($cyp_factor) ?>'>
-  </td>
- </tr>
-
- <tr<?php if (!related_codes_are_used()) echo " style='display:none'"; ?>>
```
Comment (sunsetsystems) [inline interface/patient_file/encounter/superbill_custom_full.php]: Are the related codes handled in your new stuff?

File: interface/patient_file/encounter/superbill_custom_full.php [hunk 2]
```diff
@@ -13,662 +13,250 @@
...
-
-if (isset($mode)) {
-  $code_id    = empty($_POST['code_id']) ? '' : $_POST['code_id'] + 0;
-  $code       = $_POST['code'];
-  $code_type  = $_POST['code_type'];
-  $code_text  = $_POST['code_text'];
-  $modifier   = $_POST['modifier'];
-  $superbill  = $_POST['form_superbill'];
-  $related_code = $_POST['related_code'];
-  $cyp_factor = $_POST['cyp_factor'] + 0;
```
Comment (sunsetsystems) [inline interface/patient_file/encounter/superbill_custom_full.php]: Is CYP Factor handled?

File: interface/patient_file/encounter/superbill_custom_full.php [hunk 3]
```diff
@@ -13,662 +13,250 @@
...
-$reportable = 0;
-$financial_reporting = 0;
-
-if (isset($mode)) {
-  $code_id    = empty($_POST['code_id']) ? '' : $_POST['code_id'] + 0;
-  $code       = $_POST['code'];
-  $code_type  = $_POST['code_type'];
-  $code_text  = $_POST['code_text'];
-  $modifier   = $_POST['modifier'];
-  $superbill  = $_POST['form_superbill'];
```
Comment (sunsetsystems) [inline interface/patient_file/encounter/superbill_custom_full.php]: "superbill" here is the group name, if applicable.  Is that handled?

File: interface/patient_file/encounter/superbill_custom_full.php [hunk 4]
```diff
@@ -13,662 +13,250 @@
...
-?>
-</script>
-
-</body>
-</html>
+  <head>
+    <?php html_header_show(); ?>
+    <link rel="stylesheet" href="<?php echo attr($css_header);?>" type="text/css">   
+    <link rel="stylesheet" href="../../../public/assets/font-awesome-4-6-3/css/font-awesome.css" type="text/css">      
+    <script type="text/javascript" src="../../../public/assets/jquery-min-1-7-2/index.js"></script>
```
Comment (Wakie87) [inline interface/patient_file/encounter/superbill_custom_full.php]: Need to use new `<?php echo $GLOBALS['assets_static_relative']; ?>`

File: interface/patient_file/encounter/superbill_custom_full.php [hunk 5]
```diff
@@ -13,662 +13,250 @@
...
-</script>
-
-</body>
-</html>
+  <head>
+    <?php html_header_show(); ?>
+    <link rel="stylesheet" href="<?php echo attr($css_header);?>" type="text/css">   
+    <link rel="stylesheet" href="../../../public/assets/font-awesome-4-6-3/css/font-awesome.css" type="text/css">      
+    <script type="text/javascript" src="../../../public/assets/jquery-min-1-7-2/index.js"></script> 
+    <script type="text/javascript" src="../../../public/assets/jquery-fancybox-1-2-6/index.js"></script>
```
Comment (Wakie87) [inline interface/patient_file/encounter/superbill_custom_full.php]: Need to use new <?php echo $GLOBALS['assets_static_relative']; ?>

File: interface/patient_file/encounter/superbill_custom_full.php [hunk 6]
```diff
@@ -13,662 +13,250 @@
...
+    <link rel="stylesheet" type="text/css" href="../../../public/assets/jquery-fancybox-1-2-6/index.css" media="screen" />
```
Comment (Wakie87) [inline interface/patient_file/encounter/superbill_custom_full.php]: Need to use new <?php echo $GLOBALS['assets_static_relative']; ?>

**Comment by robertdown** [comment]
Closing this since outdated PR

**Pull request closed without merging**